samples/bluetooth/hci_uart_async/prj.conf

```diff
@@ -0,0 +1,25 @@
+# Not implemented yet in hci_uart_async
+#CONFIG_BT_CTLR_ASSERT_HANDLER=y
+
+# hci_uart_async
+CONFIG_SERIAL=y
+CONFIG_UART_ASYNC_API=y
+
+# hci_raw (dependency of hci_uart)
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_HCI_RAW_H4=y
+CONFIG_BT_HCI_RAW_H4_ENABLE=y
+
+# Controller configuration. Modify these for your application's needs.
+CONFIG_BT_MAX_CONN=16
+CONFIG_BT_BUF_ACL_RX_SIZE=255
+CONFIG_BT_BUF_CMD_TX_SIZE=255
+CONFIG_BT_BUF_EVT_DISCARDABLE_SIZE=255
+
+# Send an initial HCI_Command_Complete event on boot without waiting for
+# HCI_Reset.
```

Comment:
can you add: `Needs to be enabled in the host application too.`

Reply:
I'll add it. It needs to because it will log errors in the Host otherwise right?

There is wording in the spec that suggests that the Host must always accept these.

> The Controller can send an HCI_Command_Complete event with Command Opcode 0x0000 at any time [...]
>
> -- Core 5.4 E 7.7.14

Reply:
yeah it will log a warning I think. What I wanna say is that this config has to be set on both MCUs for the feature to work properly (waiting for the NOP on bt_init).